fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -614,4 +617,37 @@ private static URI checksumURI( final String checksum ) {
         }
         return null;
     }
+
+    protected void createFixityEvent(final EventBus eventBus, final RdfStream rs,
+                                     final String baseURL, final String agent,
+                                     final String userID) {
+        final FixityEvent fixityEvent = new FixityEvent();
+        fixityEvent.setDate(System.currentTimeMillis());
+        fixityEvent.setType(128);
```

Comment:
Ideally, we would post the JMS messages to the eventBus directly and avoid having to create an int mask for the event filtering. But if we do need to do this, starting much higher, like 4096 would be better to avoid possibly conflicting with future event types added to JCR.